webpack.config.js

```diff
@@ -10,7 +10,18 @@ module.exports = {
         filename: './mpw.js',
         library: 'MPW',
         libraryTarget: 'var',
-	clean: true,
+	    clean: true
+    },
+    devServer: {
+        static: {
+          directory: path.join(__dirname, './'),
```

Comment:
`./dist` is the only directory that needs hosting (since that's the 'compiled' app output), we should host dist rather than the root directory, right?

Reply:
I think this is a watch directory. When i change that to `./dist/` it will not hot reload. Try it yourself ;)

Reply:
Will try once tested, but I'd imagine after the 'build' finishes it should be fine (as the dist files would update then).

Reply:
From testing it seems you're right, the root hosted `npm run dev` loads the compiled output, interesting haha, I'll dismiss this. :rocket: